pkg/users/accounts-list.js

```diff
@@ -129,7 +105,16 @@ const getGroupRow = (group, accounts) => {
     const columns = [
         {
             sortKey: group.name,
-            title: <Flex alignItems={{ default: 'alignItemsCenter' }}><div className={"dot " + groupColorClass} /><FlexItem>{group.name}</FlexItem></Flex>,
+            title: (
+                <Flex alignItems={{ default: 'alignItemsCenter' }}>
+                    <div className={"dot " + groupColorClass} />
+                    <FlexItem>
+                        <Button isInline variant="link" onClick={() => cockpit.jump("/users#/group/" + group.name)}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230222-131216-482495e4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)